action.yml

```diff
@@ -0,0 +1,41 @@
+name: "Foresight Test Kit"
+description: "Test and Coverage ultimate analyzer"
+author: "Thundra"
+inputs:  
+  github_token:
+    description: "GitHub API Access Token"
+    default: ${{ github.token }}
+    required: false
+  api_key: 
+    description: "Foresight Api Key"
+    required: true
+  test_framework:
```

Comment:
Are we really getting test framework name or test report format?

Reply:
Do we need an input name update as we did for coverage (`coverage_framework` => `coverage_format`)?